JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -580,19 +580,24 @@ module Microsoft.ApplicationInsights {
         /**
         * Checks if a request url is not on a excluded domain list and if it is safe to add correlation headers
         */
-        public static canIncludeCorrelationHeader(config: IConfig, requestUrl: string) {
+        public static canIncludeCorrelationHeader(config: IConfig, requestUrl: string, currentHost: string) {
             if (config && config.disableCorrelationHeaders) {
```

Comment:
is this a global setting for enabling/not enabling correlation. and enableCorsCorrelation is for cross domain specificially?

Reply:
defined everything in the issue as Sergey requested - Create separate configuration setting for enabling CORS correlation #594